demos/main/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
       android:banner="@drawable/ic_banner"
       android:largeHeap="true"
       android:allowBackup="false"
+      android:usesCleartextTraffic="true"
```

Comment:
If we proceed with this pull request, I think the cleartext traffic and `media.exolist` changes should be reverted.

Reply:
Acknowledged.

Reply:
Please could you revert these files and synch to `dev-v2` HEAD and we can try to get this merged. Thanks!

Reply:
OK sure.